src/Shuffler/Shuffler.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Shuffler } from './Shuffler.js';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+
+describe("Shuffler", () => {
```

Comment:
Will need to come back to this to add integration tests when shuffle functions are made.